transmart-core-db/grails-app/domain/org/transmartproject/db/metadata/DimensionDescription.groovy

```diff
@@ -100,6 +100,6 @@ class DimensionDescription {
      * @return a list of all dimensions
      */
     static List<Dimension> getAllDimensions() {
-        findAll { name != LEGACY_MARKER }*.dimension
+        findAll { name != DimensionDescription.LEGACY_MARKER }*.dimension
```

Comment:
I know, for some reason it is needed here, it fails without it. (though IntelliJ also thinks it is not needed)